generator/templates/server/urlbuilder.gotmpl

```diff
@@ -139,7 +139,7 @@ func ({{ .ReceiverName }} *{{ pascalize .Name }}URL) SetBasePath(bp string) {
 
 // Build a url path and query string
 func ({{ .ReceiverName }} *{{ pascalize .Name }}URL) Build() (*url.URL, error) {
-  var result url.URL
+  var _result url.URL
```

Comment:
I think it's better to make a combination here or ensure that the identifier is unique, has some randomness to it.
Otherwise we'll end up in the same situation as before because the magical string has changed from result to _result

Reply:
problem with randomness is that generation is no more stable, which is another problem...

Reply:
for my part, the `_result` is good enough. It's no silver bullet, but reduces the likelyhood of conflict.

Reply:
The `_result` also matches `_path` and `_basePath` in the function.  If a combination (nothing random please) were used, such as `{{ .ReceiverName }}_{{ . Name }}_result` then it should be added to those other variables as well.

Reply:
yes sure, I just meant not `s/result/_result/` something that has a much higher likelihood of being unique

Reply:
@fredbi released new bindata.go, so what we will do about _result? I agree that if we change this in some manner, other variables should be changed also in same manner. I thought that variables named starting with _ deliberately to avoid conflicts. May be should forbid any property name starting with _ ? Any way it is a bad practice to use such names.